README.md

```diff
@@ -41,3 +41,17 @@ $ bundle exec rubocop
 ```
 
 You do not need to include rubocop directly in your application's dependencies. Cw-style will include a specific version of `rubocop` and `rubocop-rspec` that is shared across all projects.
+
+## Gem Versioning
+- The gem version needs to be bumped along with any changes to the code.
+  - Manually bump the version number in [version.rb](https://github.com/charitywater/cw-style/blob/master/lib/charity_water/style/version.rb)
+- The gemfury-hosted gem is automatically updated when changes to this repo are merged.
+- Apps that include this gem as a dependency need to be updated to use the latest version of the gem.
+  - When making changes to this repo, be sure to update the gem version in all apps that use this gem by running `bundle update cw-style` and commiting the Gemfile.lock changes.
+  
+Apps that use this gem:
```

Comment:
nb, this search might be preferable: https://github.com/search?utf8=%E2%9C%93&q=org%3Acharitywater+%22cw-style%22+extension%3Ayml&type=Code&ref=advsearch&l=&l=

Or to the lock file, thus showing the current version: https://github.com/search?utf8=%E2%9C%93&q=org%3Acharitywater+%22cw-style%22+extension%3Alock&type=Code&ref=advsearch&l=&l=